mLRS/Common/hal/esp/tx-hal-generic-2400-pa-esp8285.h

```diff
@@ -0,0 +1,127 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+//*******************************************************
+// hal
+//********************************************************
+
+//-------------------------------------------------------
+// ESP8285, ELRS GENERIC 2400 PA TX, can only do SiK
+//-------------------------------------------------------
+
+#define DEVICE_HAS_SINGLE_LED
+//#define DEVICE_HAS_JRPIN5
+//#define DEVICE_HAS_IN
+#define DEVICE_HAS_NO_COM
+#define DEVICE_HAS_NO_DEBUG
+//#define DEVICE_HAS_NO_SERIAL
+
+// #define USE_FEATURE_MAVLINK_PARAMS // has no CLI, no Lua, hence needs this
```

Comment:
Should this be defined here?  Currently it is commented out?

Reply:
that's for the moment
I'm not yet sure how we should do that, if it should be available for all modules, just those which can't have oled,cli,jrpin5, or what ...